base_classes/NXcsg.nxdl.xml

```diff
@@ -30,13 +30,13 @@
 	   xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
 	   >
   <doc>
-    Constructive Solid Geometry base class, using :ref:`NXquadric` and :ref:`NXoff_geometry`</doc>
+    Constructive Solid Geometry base class</doc>
```

Comment:
State it combines the definition of leaf and branching nodes of a csg tree.

Reply:
Implemented